pkg/status/status_otlp.go

```diff
@@ -8,24 +8,39 @@
 package status
 
 import (
-	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"fmt"
+	"io"
+	"net/http"
+
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/otlp"
 )
 
 // GetOTLPStatus parses the otlp pipeline and its collector info to be sent to the frontend
 func GetOTLPStatus() map[string]interface{} {
-	status := make(map[string]interface{})
-	otlpIsEnabled := otlp.IsEnabled(config.Datadog)
-	var otlpCollectorStatus otlp.CollectorStatus
-	if otlpIsEnabled {
-		otlpCollectorStatus = otlp.GetCollectorStatus(common.OTLP)
-	} else {
-		otlpCollectorStatus = otlp.CollectorStatus{Status: "Not running", ErrorMessage: ""}
+	if !otlp.IsEnabled(config.Datadog) {
+		return map[string]interface{}{
```

Comment:
We used to always return `otlpCollectorStatusErr` while we don't anymore, is that okay?

Reply:
Yeah, the template simply doesn't display it if it's not there.